src/pal/inc/rt/unknwn.h

```diff
@@ -34,6 +34,17 @@ IUnknown
     virtual ULONG STDMETHODCALLTYPE AddRef( void) = 0;
         
     virtual ULONG STDMETHODCALLTYPE Release( void) = 0;
+    template<class Q>
+            HRESULT
+#ifdef _M_CEE_PURE
```

Comment:
This is how it is defined in windows headers. I couldn't find any information on _M_CEE_PURE. if you want I can remove it. Let me know.